src/player.cpp

```diff
@@ -1039,7 +1060,7 @@ std::string Player::GetEncoding() {
 
 int Player::GetSpeedModifier() {
 	if (Input::IsPressed(Input::FAST_FORWARD)) {
-		return Input::IsPressed(Input::PLUS) ? 10 : speed_modifier;
+		return Input::IsPressed(Input::PLUS) ? 10 : 3;
```

Comment:
Not possible since we hook into this from the android app.

Reply:
Fixed